### DISCUSSION TITLE
Guest (not logged in) users no longer able to use "visit" button to view deployment output

### DISCUSSION THREAD

**Discussion by e-im** [vercel/community#3687] [Bug Report]
### Page to Investigate

https://vercel.com/papermc/docs/7GyVFeXMkiDEZW95zfokuuy9SgMD

### Steps to Reproduce

1. Click "Visit" button (top right).
2. See that you need to authenticate.

### Additional Information

On projects where the source and deployment are publicly viewable, you are no longer able to use the "visit" button to show the build output.

Rather than linking directly to the deployment, it now links to `https://vercel.live/link/docs-b7o6ivusi-papermc.vercel.app?via=deployment-visit-button` (in the example above) which requires authentication.

"Vercel Authentication" or any sort of security/deployment protection is disabled on the project.

Other links in the deployment page continue to link to the preview (e.g. clicking on the screenshot of the preview)

**Comment by amyegan** [comment]
Thanks for reporting this! I sent it to our Website team for investigation

**Comment by e-im** [comment]
This seems to have been resolved. The example above (when viewing without an account) now links directly rather than through `vercel.live`.

Thanks
